src/utils/helpers.coffee

```diff
@@ -64,6 +65,15 @@ exports.excludeFile = (fileName, options) ->
             # Only instrument files that are inside the project.
             excluded = true
 
+        # For each exclude value try to use it as a pattern to exclude files
+        exclude.map (pattern) ->
+            glob.sync pattern,
+                dot: true
```

Comment:
Without `dot: true`, patterns that should normally include files starting with `.` would not be included. That is fine if you are using patterns to include files, but if you use patterns to exclude files you usually want the dot files to be excluded, too.